README.md

```diff
@@ -10,7 +10,10 @@ Instead of manually calling `use()` for every request, `use()` is called automat
 
 ```js
 
-var superagent = require('superagent-use');
+var superagent = require('superagent');
+/* Pass the superagent object to superagent-use
+   in order to be provided with new `use` functionality */
+require('superagent-use')(superagent);
```

Comment:
Not really, apart from the fact that other superagent plugins work the same way, it doesn't seem possible. [Here](https://github.com/visionmedia/superagent/blob/master/lib/node/index.js#L33) is how the exported `request` is instantiated, and as you can see [here](https://github.com/visionmedia/superagent/blob/master/lib/node/index.js#L917) the object is provided with some additional methods.